Algorithm.Python/HistoryAlgorithm.py

```diff
@@ -108,16 +108,16 @@ def Initialize(self):
 
         # we can also access the return value from the multiple symbol functions to request a single
         # symbol and then loop over it
-        singleSymbolQuandl = allQuandlData.loc["CHRIS/CME_SP1"]
-        self.AssertHistoryCount("allQuandlData.loc[\"CHRIS/CME_SP1\"]", singleSymbolQuandl, 250)
+        singleSymbolQuandl = allQuandlData.loc["CHRIS/CME_SP1.QuandlFuture"]
```

Comment:
I think the only breaking change here is that `PandasData` is using `_symbol.ToString()`-> which ends up fetching the `"CHRIS/CME_SP1.QuandlFuture"` from the `SymbolCache` at https://github.com/QuantConnect/Lean/blob/master/Common/Python/PandasData.cs#L231 which breaks the `.loc["CHRIS/CME_SP1"]` -> wonder if we could add some work around to this 🤔 
Calls like `self.Securities["CHRIS/CME_SP1"]` and `self.History(QuandlFuture, "CHRIS/CME_SP1", timedelta(365))` seem to be working correctly

Reply:
Hmm... pandas data is using `_symbol.ToString()` -- thanks for that clue! I noticed that the `string` -> `Symbol` implicit conversion was broken for custom data as noted in the PR description. I'll investigate the referenced code to see if we can get it to attempt to resolve via the `SymbolCache`'s new heuristic, allowing resolution of old ticker-only custom symbol strings.